### PR TITLE
Document setting (via config) the limit of maximum devices (deviceauth)

### PR DESCRIPTION
**Pull request by PiotrPrzybylak**
Issues: MEN-1514

Changelog: None

Signed-off-by: Piotr Przybylak <piotr.przybylak@rndity.com>

### CODE REVIEW

File: 06.Administration/02.Production-installation/docs.md [hunk 1]
```diff
@@ -498,6 +498,22 @@ you can supply a whitespace-separated list of hostnames as follows:
...
+Locate the `mender-device-auth` service.
+Add `enviroment` section if absent. In `enviroment` section add `MAX_DEVICES_LIMIT_DEFAULT` variable with integer value. `0` represents `no limit` and is the default.
+The updated entry should look like this:
+
+```yaml
+    ...
+    mender-device-auth:
+        ...
+        environment:
+            MAX_DEVICES_LIMIT_DEFAULT: 15
```
Comment (bboozzoo) [inline 06.Administration/02.Production-installation/docs.md]: I believe it's DEVICEAUTH_MAX_DEVICES_LIMIT_DEFAULT

File: 06.Administration/02.Production-installation/docs.md [hunk 2]
```diff
@@ -498,6 +498,22 @@ you can supply a whitespace-separated list of hostnames as follows:
 ```
 
 
+#### Device Authentication Service
+
+Locate the `mender-device-auth` service.
+Add `enviroment` section if absent. In `enviroment` section add `DEVICEAUTH_MAX_DEVICES_LIMIT_DEFAULT` variable with integer value. `0` represents `no limit` and is the default.
```
Comment (drewmoseley) [inline 06.Administration/02.Production-installation/docs.md]: environment is misspelled

I suggest wording this "Add the \`environment\` section is it is absent.  In the \`environment\` section, add the \`DEVICE_AUTH_MAX_DEVICES_LIMIT_DEFAULT\` variable with an integer value. ...

**Comment by drewmoseley** [comment]
I see that this is merged already.  I'll submit my requested changes in a follow-on PR

**Comment by drewmoseley** [comment]
@PiotrPrzybylak does this need to be backported to any earlier branch or is it master-only?

**Pull request merged**